tests/operators/test_load_file.py

```diff
@@ -40,16 +41,17 @@
 )
```

Comment:
Do we have a test case covering the scenario: https://github.com/astronomer/astro-sdk/issues/366?

Reply:
Do we need that? we have every load_file() test using that interface.